include/deal.II/lac/pointer_matrix.h

```diff
@@ -583,8 +583,7 @@ new_pointer_matrix_base(const TridiagonalMatrix<numberm> &matrix, const Vector<n
 
 template <typename VectorType>
 inline
-PointerMatrixBase<VectorType>::~PointerMatrixBase ()
-{}
+PointerMatrixBase<VectorType>::~PointerMatrixBase () = default;
```

Comment:
Same issue here, I assume?

Reply:
No, that was an oversight of mine. I now rechecked the header files.